lib/delayed/syck_ext.rb

```diff
@@ -36,7 +36,11 @@ def self.yaml_tag_read_class(name)
 module YAML
   def load_dj(yaml)
     # See https://github.com/dtao/safe_yaml
-    # When the method is there, we need to load our YAML like this...
-    respond_to?(:unsafe_load) ? load(yaml, :safe => false) : load(yaml)
+    # When the module is there, we need to load our YAML like this...
+    if Object.const_defined?(:SafeYAML)
+      load(yaml, :safe => false)
+    elsif respond_to?(:unsafe_load) # psych >= 3.3.2
+      unsafe_load(yaml)
+    end
```

Comment:
Is it intentional that this might now return nil (if both SafeYAML and unsafe_load aren't defined) ?  I don't know enough about psych/safeyaml, but seems like this should either fall back to `load(yaml)`, or maybe throw an exception to highlight the problem

Reply:
Thanks for your review! 

I'm going to revert this change because I noticed that this code is loaded when "syck" is used, and has nothing to do with the current YAML (pych). syck is not available in Ruby 2.0 or later, so I guess I could just delete this code itself, but I think that's outside the scope of this PR, so I'll just undo it.